### PR TITLE
[docs] cosmetic edits sharding etc

### PR DESCRIPTION
**Pull request by ddhodge**
Various minor edits and corrections to 3 files (#9010)
Added fix for #9150

### CODE REVIEW

**Review by srinivasa-vasu — COMMENTED** [review]
LGTM

File: docs/content/latest/architecture/docdb-sharding/sharding.md [hunk 1]
```diff
@@ -34,13 +34,11 @@ With (consistent) hash sharding, data is evenly and randomly distributed across
...
 
 ![tablet_hash](/images/architecture/tablet_hash.png)
 
-The insert/update/upsert by the end user is processed by serializing and hashing the primary key into byte-sequences and determining the tablet they belong to. Let us assume that the user is trying to insert a key k with a value v into a table T. The figure below illustrates how the tablet owning the key for the above table is determined.
+The insert/update/upsert by the end user is processed by serializing and hashing the primary key into byte-sequences and determining the tablet they belong to. Assume that a user is trying to insert a key k with a value v into a table T. The figure below illustrates how the tablet owning the key for the above table is determined.
```
Comment (lizayugabyte) [inline docs/content/latest/architecture/docdb-sharding/sharding.md:41]: The figure below illustrates how the tablet owning the key for the above table is determined. --> The following illustration demonstrates how the tablet owning the key for the above table is determined:

File: docs/content/latest/explore/transactions/explicit-locking.md [hunk 2]
```diff
@@ -26,96 +26,85 @@ showAsideToc: true
...
+
 * `FOR UPDATE`
 * `FOR NO KEY UPDATE`
 * `FOR SHARE`
 * `FOR KEY SHARE`
 
-Let us look at en example with the `FOR UPDATE` row lock, where we first `SELECT` a row for update to lock it and subsequently update it. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked. To try out this scenario, first create an example table with sample data as shown below.
+The following example uses the `FOR UPDATE` row lock. First, a row is selected for update, thereby locking it, and subsequently updated. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked. 
+
+To try out this scenario, first create an example table with sample data as shown below.
```
Comment (lizayugabyte) [inline docs/content/latest/explore/transactions/explicit-locking.md:46]: To try out this scenario, first create an example table with sample data as shown below. -->  To try out this scenario, first create an example table with sample data, as follows:

**Review by lizayugabyte — COMMENTED** [review]
Left a few minor comments.

**Pull request merged**